plugins/broker-redis/src/constants.ts

```diff
@@ -1,49 +1,58 @@
-import { CreateServerDockerComposeDevParams, CreateServerDockerComposeParams } from "@amplication/code-gen-types";
+import {
+  CreateServerDockerComposeDevParams,
+  CreateServerDockerComposeParams,
+} from "@amplication/code-gen-types";
 import { join } from "path";
 
 export const staticsPath = join(__dirname, "static");
 
 export const templatesPath = join(__dirname, "templates");
 
 export const dependencies = {
-    dependencies: {
-        "redis": "3.1.2",
-        "@nestjs/microservices": "8.2.3",
-        "rxjs": "^7.8.1"
-    }
-}
+  dependencies: {
+    redis: "3.1.2",
+    "@nestjs/microservices": "8.2.3",
```

Comment:
8.2.3?

Reply:
this PR only run the linting rules and not update the version yet